README.md

```diff
@@ -91,16 +103,26 @@ enum TaskStatus: int
     case COMPLETED = 1;
     case CANCELED = 2;
 }
+
+enum Role
+{
+    use Names;
+
+    case ADMINISTRATOR;
+    case SUBSCRIBER;
+    case GUEST;
+}
 ```
 
 #### Use the `names()` method
 ```php
 TaskStatus::names(); // ['INCOMPLETE', 'COMPLETED', 'CANCELED']
+Role::names(); // ['ADMINISTRATOR', 'SUBSCRIBER', 'GUEST']
 ```
 
 ### Values
 
-This helper returns a list of case *values* in the enum.
+This helper returns a list of case *values* in the enum. _**NB:** pure enums don't have values, so this will only ever return an empty array._
```

Comment:
I'm in two minds about this - the indices don't _technically_ exist for pure enums, and the "name" is _technically_ the value. I decided to make it return an empty array to make the behaviour "distinct" from `names()`, but honestly I think that it should probably just return the list of names instead of the list of values. I'm also unsure if using the index is "correct" in #7 (and as such, the `from()`/`tryFrom()` should mirror `fromName()`/`tryFromName()`, or otherwise not exist)

Reply:
Maybe we can get rid of the indexes completely then and always use the name, when working with pure enums?